Milestone 1: Adding Description About Local Setups in Readme.md file

bushmusi opened the pull request:
# Linters milestone 1: Setups

***In this milestone 1:***
    1. I have added which setups are needed in local env
    2. Update Readme file

bushmusi committed:
Add local needed setups in readme.md

Whoistolu reviewed (changes requested):
Hi @bushmusi 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

### To Highlight
- [ ] Kindly make sure that your Pull Request has a descriptive title that describes the activity performed what changes are inside. Your current PR title reads 'Linters'.
- [ ] It will be nice to have a descriptive BRANCH name (e.g. 'created_sidebar' or 'added_comments_section'). This ensures proper github flow
- [ ] Kindly make sure that your `.html file`, `.css file` are in the branch you made the Pull request from.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

bushmusi commented:
Hi @Whoistolu 

It was a great comment, you have mentioned main things I have forgotten to do. But it does not mean keep commenting since I am student I will improve it in next project 🤣 

# New Update 
1. I have changed my PR title and description
2. I have updated branch name which is descriptive
3. I make sure HTML and CSS files are current branch

bushmusi commented:
Hi @Whoistolu 

I am back again 🦘. After I change the branch name as per your comment the PR closed by itself. Can you review the new one with this [link](https://github.com/bushmusi/Hello-Microverse/pull/7)
